<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FContractorEdit</class>
 <widget class="QDialog" name="FContractorEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contractor edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/hacc.qrc">
    <normaloff>:/items/contractor.png</normaloff>:/items/contractor.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="ui::icons::WIconSelect" name="pwIcons" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leName"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbClearName">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/hacc.qrc">
           <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbOwnAccount">
     <property name="text">
      <string>Own account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lAccounts">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Accounts</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lAccountsLayout"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::icons::WIconSelect</class>
   <extends>QWidget</extends>
   <header>wiconselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>cbOwnAccount</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tbClearName</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FContractorEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FContractorEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
